todoApp/spec/main.tsp

```diff
@@ -275,9 +287,18 @@ namespace TodoItems {
 
     @sharedRoute
     @post
-    op createAttachment(
+    op createUrlAttachment(
+      @header contentType: "application/json",
+      @path itemId: TodoItem.id,
+      @body contents: TodoUrlAttachment,
+    ): WithStandardErrors<NoContentResponse | NotFoundResponse>;
+
+   @sharedRoute
+    @post
+    op createFileAttachment(
+      @header contentType: "multipart/form-data",
       @path itemId: TodoItem.id,
-      @body contents: TodoAttachment,
-    ): WithStandardErrors<NoContentResponse | NotFoundErrorResponse>;
+      @multipartBody body: FileAttachmentMultipartRequest,
+    ): WithStandardErrors<NoContentResponse | NotFoundResponse>;
```

Comment:
NotFoundResponse would be a success response, would it be expected?

Reply:
So we shall still keep `NotFoundErrorResponse`